config/config-network.yaml

```diff
@@ -49,4 +49,4 @@ data:
   # For more information, visit
   # https://istio.io/docs/tasks/traffic-management/egress/
   #
-  istio.sidecar.includeOutboundIPRanges: "*"
+  istio.sidecar.includeOutboundIPRanges: "10.0.0.1/24"
```

Comment:
This change was probably unintentionally included with this PR. I have the same local change and have almost accidentally included it in a PR as well :smile:

Reply:
Whoops. That's embarrassing. Thank for catching it, @bbrowning.